client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -400,19 +404,13 @@ class ListItem extends PureComponent {
       },
       isMember
     } = this.props;
-    const {
-      areDetailsVisible,
-      disableToggleButton,
-      done,
-      isNameEdited
-    } = this.state;
+    const { areDetailsVisible, disableToggleButton, isNameEdited } = this.state;
     const isEdited = nameLock || descriptionLock;
 
     return (
       <li
         className={classNames('list-item', {
-          'list-item--restore': !done && isOrdered,
-          'list-item--done': done || isOrdered,
+          'list-item--done': isOrdered,
```

Comment:
Shouldn't it be named `done`?

Reply:
On backend `isOrdered`, here was `done` - it was a bit confusing, so I decided to make it consistent and just use `isOrdered` everywhere

Reply:
Ok, however, `done` is better IMHO as the `isOrdered` could be confused as if the item is sorted or not. Anyway, that would require a migration.

Reply:
hmm, maybe we should think about some refactoring? And change isOrdered to isDone which is more universal... (in the separate task of course)

Reply:
added as new taski in JIRA